tests/api.py

```diff
@@ -629,3 +652,57 @@ def api_get_backend():
     assert backend["is_default"] == True
     assert backend["caps"]
     pprint.pprint(backend)
+
+
+def _pixel_diff(a: bytes, b: bytes, threshold=1) -> bool:
+    for x, y in zip(a, b):
+        if abs(x - y) > threshold:
+            return True
+    return False
+
+
+def api_media_change():
```

Comment:
Can we move this test to the media tests and register the live controls so they are available in ngl-viewer, it will remove the need to craft that kind of scene to exercise this scenario with other medias.

Reply:
Moved to `live.py` instead and used the keyframes callback mechanism.

I can't add the live control easily though (it would require to add a `live_id` to the `Media` node, which raises question about a common path field for other nodes with a filepath based parameter)

Reply:
Thanks.